src/fullTimeStatus.js

```diff
@@ -0,0 +1,14 @@
+import Moment from 'moment';
+import { extendMoment } from 'moment-range';
+const moment = extendMoment(Moment);
+
+export const FTE_START_DATES = [
```

Comment:
I like this iteration, but in the long term, I think I want a way to set these values in the deployment state (ENV, DB/Redis/Company Information Store Service, SOMETHING)